webapp/src/shared/components/Row/Row.tsx

```diff
@@ -0,0 +1,54 @@
+import Icon, { IconVariant, IconSize } from '../Icon/Icon';
+import './Row.css';
+import { SmallAvatar, AvatarProps } from '../Avatar/Avatar';
+import { TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+
+type RowProps = {
+  iconVariant?: IconVariant;
+  onClick?: () => void;
+  avProps?: AvatarProps;
+  children?: string[];
+};
+
+export default function Row({
+  iconVariant,
+  onClick,
+  avProps,
+  children,
+}: RowProps) {
+  return (
+    <button className={`row paragraph-regular`} onClick={onClick}>
+      <div className={`row_wrapper`}>
+        {avProps && <SmallAvatar {...avProps} />}
+        {children && (
+          <div className={`row_text_wrapper`}>
+            {children[0] && (
+              <Text
+                variant={TextVariant.PARAGRAPH}
+                weight={TextWeight.MEDIUM}
```

Comment:
I think title is not a medium paragraph but a bold subheading, please review text sizes/types from Figma -> https://www.figma.com/file/efdjuSsiJqDyZbOxs9UTI7/App?node-id=159%3A32

Reply:
Please, read above.

Reply:
There is room for negotiation in the weight part of this style, but it's tightly coupled to size, and as long as you don't agree that your opinion, as written above, on the text size is wrong, we are blocked.